src/containers/pages/IRS/JurisdictionsReport/index.tsx

```diff
@@ -172,18 +173,18 @@ const JurisdictionReport = (props: GenericJurisdictionProps & RouteComponentProp
 
   const currLevelData = data.filter(el => el.jurisdiction_parent_id === jurisdictionId);
 
-  let columnsToUse = IRSTableColumns[SUPERSET_IRS_REPORTING_JURISDICTIONS_COLUMNS];
+  let columnsToUse = get(IRSTableColumns, SUPERSET_IRS_REPORTING_JURISDICTIONS_COLUMNS, null);
   if (currLevelData && currLevelData.length > 0) {
     if (
       currLevelData[0].jurisdiction_depth === +SUPERSET_IRS_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL
     ) {
-      columnsToUse = IRSTableColumns[SUPERSET_IRS_REPORTING_FOCUS_AREAS_COLUMNS];
+      columnsToUse = get(IRSTableColumns, SUPERSET_IRS_REPORTING_FOCUS_AREAS_COLUMNS, null);
     }
   }
 
   const tableProps = {
+    ...(columnsToUse && { columns: columnsToUse }),
```

Comment:
In case `columnsToUse` is null we wont have a `columns` prop.  Drilldown table will then generate columns from the data